examples/capi-quick-start/docker-cluster.yaml

```diff
@@ -24,6 +24,7 @@ spec:
           addons:
             cni:
               provider: calico
+              value: {}
```

Comment:
Added by mistake? I don't see `value` defined in https://github.com/d2iq-labs/capi-runtime-extensions/blob/af46151552bcdb8a8c35166b819d0ddc1aee99c2/api/v1alpha1/clusterconfig_types.go#L340-L360